src/components/Home.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable max-len */
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { v4 as uuidv4 } from 'uuid';
+import Card from 'react-bootstrap/Card';
+import Row from 'react-bootstrap/Row';
+import Container from 'react-bootstrap/Container';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import Col from 'react-bootstrap/Col';
+import { faCircleRight } from '@fortawesome/free-solid-svg-icons';
+import Map1 from '../assets/images/map1.svg';
+import Map2 from '../assets/images/map2.svg';
+import { fetchData } from '../redux/home/homeSlice';
+import TopCard from './TopCard';
+import LoadingSpinner from './Spinner';
+import { desktopMediaQuery, mobileMediaQuery } from '../media/mediaConfig';
+
+const StyledHome = styled.div`
+  ${desktopMediaQuery}
+
+  ${mobileMediaQuery}
+`;
+
+const Home = () => {
+  const airQualityParameters = ['pm25'];
+  const dispatch = useDispatch();
+  const data = useSelector((state) => state.home.home);
+  const isLoading = useSelector((state) => state.home.isLoading);
+  const error = useSelector((state) => state.home.error);
+
+  const [isSearchVisible, setIsSearchVisible] = useState(false);
+  const [searchKeyword, setSearchKeyword] = useState('');
+
+  const filteredData = data.filter(({ measurements }) => measurements.some(({ parameter }) => airQualityParameters.includes(parameter)));
+  const filteredLocations = filteredData.filter((location) => location.location.toLowerCase().includes(searchKeyword.toLowerCase()));
+
+  const toggleSearch = () => {
+    setIsSearchVisible(!isSearchVisible);
+    if (!isSearchVisible) {
+      setSearchKeyword('');
+    }
+  };
+
+  useEffect(() => {
+    dispatch(fetchData());
+  }, [dispatch]);
+
+  useEffect(() => {
+  }, [data]);
+
+  if (isLoading) {
+    return <LoadingSpinner />;
+  }
+
+  if (error) {
+    return (
+      <p className="white-text">
+        Error:
+        {' '}
+        {error}
+      </p>
+    );
+  }
+
+  return (
+    <div>
+      <StyledHome>
+        <TopCard
+          backgroundImage={Map1}
+          location="USA"
+          views="890 Views"
+          footerText="Stats by air preassure"
+          toggleSearch={toggleSearch}
+          isSearchVisible={isSearchVisible}
+          setSearchKeyword={setSearchKeyword}
+        />
```

Comment:
- [ ] Kindly modify the intro section of your app to provide more relevant information about the project you are building. Adding an arbitrary 860 views from the USA does not fit the project you are building. You are concentrating on Air pressure, then kindly provide information related to it.

  The intro section should provide more relevant information about the project, just like the design

  ![image](https://github.com/RileyManda/air-quality/assets/36542986/0a8c0643-e252-40a1-9b81-7e745895fb61)